examples/virtual_terminal/aux_inputs/main.cpp

```diff
@@ -0,0 +1,199 @@
+#include "isobus/hardware_integration/available_can_drivers.hpp"
+#include "isobus/hardware_integration/can_hardware_interface.hpp"
+#include "isobus/isobus/can_general_parameter_group_numbers.hpp"
+#include "isobus/isobus/can_network_manager.hpp"
+#include "isobus/isobus/can_partnered_control_function.hpp"
+#include "isobus/isobus/isobus_virtual_terminal_client.hpp"
+#include "isobus/utility/iop_file_interface.hpp"
+#include "isobus/utility/system_timing.hpp"
+
+#include "console_logger.cpp"
+#include "object_pool_ids.h"
+
+#include <atomic>
+#include <csignal>
+#include <iostream>
+#include <memory>
+
+//! It is discouraged to use global variables, but it is done here for simplicity.
+static std::shared_ptr<isobus::VirtualTerminalClient> TestVirtualTerminalClient = nullptr;
+static constexpr std::uint64_t MODEL_IDENTIFICATION_CODE = 1; ///< The model identification code of 'our' input device, this should be increased when changes are made to the input(s) definitions
+
+static constexpr std::uint64_t BUTTON_CYCLIC_DELAY = 3500; ///< 1 second between button presses
+static constexpr std::uint64_t B2_CYCLIC_DELAY = 1000; ///< 3.5 seconds between slider movements
```

Comment:
I think the comments on these two values might be reversed

Reply:
Good catch! Fixed in latest commit